RadioDns-PlugIt/channels/actions.py

```diff
@@ -313,6 +317,10 @@ def channels_import(request):
                         if not re.match(r"^[a-fA-F0-9]{3}$", channel.cc):
                             errors.append("cc must be 3 characters in hexadecimal for line " + line)
 
+                    if channel.mid is not None:
```

Comment:
We can probably refactor the channel validation into one function rather than duplicate all this code.